### ISSUE TITLE
early stopping

### ISSUE THREAD

**Issue by yzliao** [fchollet/keras#93]
Does Keras support early stopping right now? I have tried to implement this feature by myself but hope to know if the library supports this functionality from underlay?

**Comment by fchollet** [comment]
What you can do is display training and testing loss and accuracy at every epoch (using the arguments validation_split, verbose, show_accuracy of the .fit method). Then you can decide for yourself what is an appropriate number of epochs.
